auto_override_force:
  - 0
  - 0
  - 19.75522417627286
auto_path_resolution: 6.911146124503305e-05